### PR TITLE
Fix listening and polling mode bugs

### LINKED ISSUE #12: NoClassDefFoundError: org/apache/kafka/common/KafkaException when on CDC listening mode
**Description:**
java.lang.NoClassDefFoundError: org/apache/kafka/common/KafkaException is thrown when a CDC source is defined with listenning mode.

**Steps to reproduce:**
1. Have a siddhi app source defined with CDC Listenning mode.

### PR DESCRIPTION
**Pull request by shehan360**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.wso2.extension.siddhi.io.cdc.util;
```
Comment (tishan89) [inline component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Please add licence

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.yaml.snakeyaml.error.YAMLException;
+import org.yaml.snakeyaml.nodes.Node;
+import java.util.HashMap;
+
+/**
+ * This class is used to read query-config.yaml
+ */
+public class MyYamlConstructor extends Constructor {
+
+    private HashMap<String, Class<?>> classMap = new HashMap<String, Class<?>>();
```
Comment (tishan89) [inline component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Declaration should be a Map

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    public MyYamlConstructor(Class<? extends Object> theRoot) {
+        super(theRoot);
+        classMap.put(QueryConfiguration.class.getName(), QueryConfiguration.class);
+        classMap.put(Database.class.getName(), Database.class);
+    }
+
+    protected Class<?> getClassForNode(Node node) {
+        String name = node.getTag().getClassName();
+        Class<?> cl = classMap.get(name);
```
Comment (tishan89) [inline component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Use a meaningful variable name

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java (hunk 1 above)
Comment (shehan360) [reply component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Added license

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java (hunk 2 above)
Comment (shehan360) [reply component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Added the changes

File: component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java (hunk 3 above)
Comment (shehan360) [reply component/src/main/java/org/wso2/extension/siddhi/io/cdc/util/MyYamlConstructor.java]: Added changes

**Pull request merged**